pupil-spa/src/app/qr-code-arrival/qr-code-arrival.component.ts

```diff
@@ -3,7 +3,7 @@ import { Router } from '@angular/router'
 import { ApplicationInsightsService } from '../services/app-insights/app-insights.service';
 
 @Component({
-  selector: 'qr-code-arrival',
+  selector: 'app-qr-code-arrival',
```

Comment:
why is this being changed?

Reply:
Lint rules : all selectors must start with 'app'.

Reply:
thanks for the heads up 👍🏼